sdk/core/azure-core/src/main/java/com/azure/core/http/HttpResponse.java

```diff
@@ -66,9 +65,9 @@
      * the `subscribeOn` and `observeOn` but should be considered a template for
      * more complex situations.
      *
-     * @return The response's content as a stream of {@link ByteBuf}.
+     * @return The response's content as a stream of {@link ByteBuffer}.
      */
-    public abstract Flux<ByteBuf> body();
+    public abstract Flux<ByteBuffer> body();
```

Comment:
I like the move to use the JDK ByteBuffer versus a third party implementation. Would make it easier for other people to extend from.